kytos/core/link.py

```diff
@@ -83,6 +84,19 @@ def make_tag_available(self, tag):
         else:
             return False
 
+    def available_vlans(self):
+        """Get all available vlans from each interface in the link."""
```

Comment:
Available_vlans on a link MUST be an intersection.

Reply:
Done.